src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.Id.MachineName.cs

```diff
@@ -3,7 +3,7 @@
     public partial class Activity
     {
         #region private
-        private string generateUniquePrefix()
+        private string generateInstancePrefix()
```

Comment:
Nit: I hadn't noticed this before, but our style is to have PascalCased method names, so this should be `GenerateInstancePrefix`.